pytest_selenium/pytest_selenium.py

```diff
@@ -94,8 +94,12 @@ def driver_kwargs(request, capabilities, chrome_options, driver_args,
         driver_path=driver_path,
         firefox_options=firefox_options,
         firefox_profile=firefox_profile,
-        host=pytestconfig.getoption('host'),
-        port=pytestconfig.getoption('port'),
+        host=pytestconfig.getoption('host',
+                                    getattr(driver, 'HOST', None),
+                                    True),
```

Comment:
According to the source code for `getoption`, passing `skip=True` would "raise pytest.skip if option does not exists or has a None value". Is this what we want?

Reply:
Yes, because we are providing a default, even if that default may be `None`.
Not setting `skip` truthy will then not give us the default. At least that's what happened to me.